### PR TITLE
[DX-1682] Release Notes 5.5.1 Dashboard

### PR DESCRIPTION
**Pull request by dcs3spp**
### For internal users - Please add a Jira DX PR ticket to the subject!
[DX-1682](https://tyktech.atlassian.net/browse/DX-1682)

---

### Preview Link

[preview](https://deploy-preview-5397--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.5)

### Description

Add dashboard release notes 5.5.1

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labeled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Added detailed release notes for Tyk Dashboard version 5.5.1, including release date, breaking changes, and deprecations.
- Included a compatibility matrix for Tyk components and third-party dependencies.
- Provided upgrade instructions and highlighted the focus on bug fixes in this release.
- Added download links for Docker images and Helm charts.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.5.md</strong><dd><code>Add release notes and details for Tyk Dashboard version 5.5.1</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.5.md

<li>Added release notes for version 5.5.1.<br> <li> Included compatibility matrix for Tyk components.<br> <li> Listed third-party dependencies and tools.<br> <li> Provided upgrade instructions and release highlights.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5397/files#diff-676ee2cfeee670a30cf0815d99ddfae1a6a09fbb4aade2e98314661e24e93a5c">+122/-5</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

[DX-1682]: https://tyktech.atlassian.net/browse/DX-1682?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dcs3spp** [comment]
@caroltyk Please check Tyk charts version associated with release notes 5.5.1 is correct and update accordingly.

After this PR has been published please create a separate PR to include a download link for Tyk Charts when released.

**Review by andyo-tyk — APPROVED** [review]
Confirmed change log (no changes - just a version bump), moved Release Highlights higher for visibility. Needs the release date and Helm links etc to be completed.

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.5.md [hunk 1]
```diff
@@ -17,10 +12,84 @@ tags: ["Tyk Dashboard", "Release notes", "v5.5", "5.5.0", "5.5", "changelog"]
 Our minor releases are supported until our next minor comes out.
 
 ---
+
+## 5.5.1 Release Notes
+
+### Release Date ?? September 2024
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.5.md]:
```suggestion
### Release Date 18 September 2024
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.5.md [hunk 2]
```diff
@@ -17,10 +12,84 @@ tags: ["Tyk Dashboard", "Release notes", "v5.5", "5.5.0", "5.5", "changelog"]
 Our minor releases are supported until our next minor comes out.
 
 ---
+
+## 5.5.1 Release Notes
+
+### Release Date 24 September 2024
```
Comment (sharadregoti) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.5.md]:
```suggestion
### Release Date 26 September 2024
```

**Review by sharadregoti — APPROVED** [review]

**Pull request merged**